### PR TITLE
Dckesler/cannoli

### PR DESCRIPTION
**Pull request by dckesler**
### Description

Adds support for Cannoli chain

### CODE REVIEW

File: apps/web/pages/[chain].tsx [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
 export const inter = Inter({ subsets: ['latin'] })
 
 interface Props {
   isOutOfCELO: boolean
+  network: Network
 }
 
-export default function Home({ isOutOfCELO }: Props) {
+const Home: NextPage<Props> = ({ isOutOfCELO, network }: Props) => {
+  const networkProper = `${network[0].toUpperCase()}${network
```
Comment (aaronmgdr) [inline apps/web/pages/[chain].tsx:20]: maybe networkCapitalized

File: apps/web/pages/[chain].tsx [hunk 2]
```diff
@@ -110,9 +119,20 @@ export default function Home({ isOutOfCELO }: Props) {
...
 }
 
-export async function getServerSideProps(): Promise<{ props: Props }> {
-  const isOutOfCELO = await isBalanceBelowPar()
+export default Home
+
+export const getServerSideProps: GetServerSideProps<Props> = async (
+  context
+) => {
+  const network = context.query.chain
```
Comment (aaronmgdr) [inline apps/web/pages/[chain].tsx:127]: sometimes querys can be Arrays (according to types)

File: apps/web/src/setup-button.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import detectEthereumProvider from '@metamask/detect-provider'
 import { useAsyncCallback } from 'react-use-async-callback'
 import Image from 'next/image'
-import { inter } from '../pages/index'
+import { inter } from '../pages/[chain]'
```
Comment (aaronmgdr) [inline apps/web/src/setup-button.tsx:4]: hey this wasnt you but it is weird to import from a page into a component file. I must have messed it up. if there is another place to import inter from would be great to change that

File: apps/web/pages/[chain].tsx (hunk 2 above)
Comment (dckesler) [reply apps/web/pages/[chain].tsx:127]: Look at the line below this

File: apps/web/src/setup-button.tsx (hunk 3 above)
Comment (dckesler) [reply apps/web/src/setup-button.tsx:4]: "Hey, while you're at it...."

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**